lib/helpers/index.js

```diff
@@ -2,4 +2,19 @@
 
 const capitalise = string => string.charAt(0).toUpperCase() + string.slice(1);
 
-module.exports = { capitalise };
+const ifEquals = (a, b, options) => {
+	if (a === b) {
+		return options.fn(this);
+	}
+	return options.inverse(this);
+};
+
+const trim = (value, string, options) => {
```

Comment:
Should this be named `replace` or something different? As `trim` nearly always means "trim whitespace from either side"